ground/src/test/java/com/google/android/ground/ui/offlineareas/OfflineAreasFragmentTest.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.ui.offlineareas
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.hasDescendant
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import com.google.android.ground.BaseHiltTest
+import com.google.android.ground.R
+import com.google.android.ground.launchFragmentInHiltContainer
+import com.google.android.ground.persistence.local.stores.LocalOfflineAreaStore
+import com.google.android.ground.util.recyclerview.atPositionOnView
+import com.sharedtest.FakeData.OFFLINE_AREA
+import dagger.hilt.android.testing.HiltAndroidTest
+import javax.inject.Inject
+import kotlinx.coroutines.test.advanceUntilIdle
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@HiltAndroidTest
+@RunWith(RobolectricTestRunner::class)
+class OfflineAreasFragmentTest : BaseHiltTest() {
+
+  lateinit var fragment: OfflineAreasFragment
+  @Inject lateinit var localOfflineAreaStore: LocalOfflineAreaStore
+
+  @Before
+  override fun setUp() {
+    super.setUp()
+    launchFragmentInHiltContainer<OfflineAreasFragment> { fragment = this as OfflineAreasFragment }
+  }
+
+  @Test
+  fun `toolbar text should be correct`() {
```

Comment:
Please use title case here and everywhere else

Reply:
Also, let's rename it to something like `Toolbar text is displayed`

Reply:
done